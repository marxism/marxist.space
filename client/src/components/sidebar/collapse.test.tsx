/* eslint-env jest */

import * as React from 'react'
import Collapse from './about'
import { create } from 'react-test-renderer'

describe('Collapse', () => {
  test('renders', () => {
    expect(
      create(
        <Collapse title="asdf">
          <span>hello</span>
        </Collapse>
      ).toJSON()
    ).toMatchSnapshot()
  })
})
