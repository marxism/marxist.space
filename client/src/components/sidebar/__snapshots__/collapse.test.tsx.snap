// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collapse renders 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-top: 16px;
  font-size: 14px;
  padding: 0 16px;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  min-width: 0;
}

.c0 a {
  -webkit-transition: color 0.1s ease;
  transition: color 0.1s ease;
  text-underline-position: under;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c0 p {
  margin-bottom: 16px;
  line-height: 1.5;
}

<article
  className="c0"
>
  <p>
    marxist.space is a
     
    <a
      href="https://prolesoft.github.io"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      ProleSoft
    </a>
     
    project. Check out our
     
    <a
      href="https://prolesoft.github.io/blog"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      developer blog
    </a>
    .
  </p>
  <p>
    This project is
     
    <a
      href="https://github.com/prolesoft/marxist.space"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      free and open source
    </a>
     
    under the
     
    <a
      href="https://github.com/prolesoft/marxist.space/blob/master/LICENSE.md"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      LGPL-3.0
    </a>
     
    license.
     
    <a
      href="https://github.com/prolesoft/marxist.space/blob/master/.github/CONTRIBUTING.md"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      Contributions
    </a>
     
    are welcome. You can also email
     
    <a
      href="mailto:marxist.space@protonmail"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      marxist.space@protonmail.com
    </a>
     
    or join the discussion on
     
    <a
      href="https://matrix.to/#/!TUENHHyVTWHKRxKiSm:matrix.org?via=matrix.org"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      Matrix
    </a>
  </p>
  <p>
    <a
      href="https://stats.uptimerobot.com/5Alqxc0QYg"
      rel="nofollow noopener noreferrer"
      target="_blank"
    >
      Status Page
    </a>
    .
  </p>
</article>
`;
